cloudformation/deepfence-managed/automated-deployment/deepfence-cloud-scanner-automated-organization-deployment.template

```diff
@@ -22,14 +22,16 @@ Parameters:
     - 'arn:aws:iam::aws:policy/SecurityAudit'
     - 'arn:aws:iam::aws:policy/ReadOnlyAccess'
 Resources:
-  IAMRole:
+  CloudScannerRole:
     Type: AWS::IAM::Role
     Properties:
       Path: /
-      RoleName: deepfence-managed-cloud-scanner-read-only-access
+      RoleName: !Join
+      - ''
+      - - !Ref 'AWS::StackName'
       AssumeRolePolicyDocument: !Join
       - ''
-      - - '{"Version":"2012-10-17","Statement":[{"Effect":"Allow","Principal":{"AWS":"arn:aws:iam::070489688603:role/deepfence-managed-cloud-scanner-role"},"Action":"sts:AssumeRole"}]}'
+      - - '{"Version":"2012-10-17","Statement":[{"Effect":"Allow","Principal":{"AWS":"arn:aws:iam::122565780891:role/EKSNodeIAMRole"},"Action":"sts:AssumeRole"}]}'
```

Comment:
what is this role used for ?

Reply:
Customers permit nodes/pods associated with this role to read resources from their cloud.
This will be changed to a role in SaaS account.